### DISCUSSION TITLE
My project is.NET Framework 4.6.2

### DISCUSSION THREAD

**Discussion by howeWang97** [fsharplang-ru/pulsar-client-dotnet#287] [Q&A]
Hello:
My project is.NET Framework 4.6.2 and I would also like to use your Pulsar.Client 3.6.2. What should I do, please
   Hope to get your reply
   thank you

**Comment by Lanayx** [comment]
Hi, you should upgrade your project to .NET 8 or higher. You can also use 2.16.0 version of library, but it is not kept up to date and only gets updates if someone works on them and creates a PR to 2.x branch.

**Reply by howeWang97** [reply]
Thank you~~~
